jenkins/libdnf-tests.sh

```diff
@@ -0,0 +1,11 @@
+echo "GIT_COMMIT:" "${GIT_COMMIT}"
+
+export BUILD_DIR='./build'
+
+cd './src/dnf-plugins/product-id'
+rm -rf "${BUILD_DIR}"
+mkdir "${BUILD_DIR}"
+cd "${BUILD_DIR}"
+cmake ../
```

Comment:
I'd simulate a "distro build" and set the installation prefix to `/usr`, so passing `-DCMAKE_INSTALL_PREFIX=/usr` to cmake

Reply:
What would be benefit of this change? We run only unit test here. We do not install this plugin to the system at all here.

When following PR will be merged: https://github.com/candlepin/subscription-manager/pull/2668 then we can do more tests and maybe we can add some integration tests of libdnf plugin too.

Reply:
But thanks for looking at this PR. I have to resolve conflicts in this PR.

Reply:
Good idea.